lib/salus.rb

```diff
@@ -55,6 +58,19 @@ def scan(
                                        ignore_config_id: ignore_config_id,
                                        cli_scanners_to_run: only)
 
+      if sarif_diff_full != ""
+        begin
+          processor.create_full_sarif_diff(sarif_diff_full)
+        rescue StandardError => e
+          puts "Failed to get sarif diff #{e.inspect}"
+          system_exit(EXIT_FAILURE)
+        end
+
+        processor.report.report_uris.select! { |u| u['format'] == FULL_SARIF_DIFF_FORMAT }
```

Comment:
Do we need a new format?  Might be nice to just reuse "sarif" as the diff is producing valid sarif.  We can probably retire the old sarif diff format at some point.  Thoughts?